From 265e406e254c8d84016b12b344d8df71d1765dd1 Mon Sep 17 00:00:00 2001
From: Jappie Klooster <jappieklooster@hotmail.com>
Date: Fri, 2 Apr 2021 16:33:18 -0400
Subject: [PATCH] Add cmake install directives

To make nix builds work, it expect a make install command to
be available.
Adding these directives seems to fix the build.

If it's no trouble to you, please consider adding them.
---
 CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cbfc9c1..948c432 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,8 @@ set(SOURCE_FILES
         uInput.cpp uInputSetup.cpp uInputResource.cpp
         uInput.hpp CommonIncludes.hpp uInputSetup.hpp)
 
+include(GNUInstallDirs)
+
 add_library(uInputPlus ${SOURCE_FILES})
 target_include_directories(uInputPlus PUBLIC .)
 
@@ -15,3 +17,9 @@ target_link_libraries(uInputPlus_test uInputPlus)
 
 configure_file(uInputPlus.pc.in uInputPlus.pc @ONLY)
 
+
+install(TARGETS uInputPlus
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(FILES uInput.hpp CommonIncludes.hpp uInputSetup.hpp
+        DESTINATION include/)
+
-- 
2.29.2

